MetaMask/metamask-extension
# Move token detection to top of assets

## Explanation

Moves the prompt for importing detected tokens from the bottom to the top of the assets view.  So that users can see it without scrolling to the bottom.

Implements https://consensyssoftware.atlassian.net/browse/MMASSETS-24

## Screenshots/Screencaps

### Before

![image](https://github.com/MetaMask/metamask-extension/assets/3500406/464be2e2-acd4-40dc-a4f0-5af469503824)

### After

![image](https://github.com/MetaMask/metamask-extension/assets/3500406/d9655ae0-6fb7-4c45-8110-d68b74526c8c)

## Manual Testing Steps

Send an ERC-20 token to the account and verify the import prompt appears at the top.

## Pre-merge author checklist


## Pre-merge reviewer checklist
